File: cpp/Baju.cpp
#include <iostream>
#include <string>
#include "Aksesoris.cpp"

using namespace std;

//deklarasi class shirt yang menjadi anak dari Aksesoris
class Baju : public Aksesoris 
{
    //atribut private
    private:
     string size;
     string merk;

    public:
    //konstruktor
    Baju()
    {
        this->size = "";
        this->merk = "";
    }
    Baju(string size, string merk)
    {
        this->size = size;
        this->merk = merk;
    }

    //setter
    void setSize(string size)
    {
        this->size = size;
    }
    void setMerk(string merk)
    {
        this->merk = merk;
    }

    //getter
    string getSize()
    {
        return this->size;
    }
    string getMerk()
    {
        return this->merk;
    }

    //destructor
    ~Baju()
    {
        
    }
};

File: python/Baju.py
#library
from Aksesoris import Aksesoris

#deklarasi class yang menjadi anak dari Aksesoris
class Baju(Aksesoris):
    #atribut private
    __size = ""
    __merk = ""

    #constructor
    def __init__(self, size = "", merk = ""):
        self.__size = size
        self.__merk = merk

    #setter
    def setsize(self,size):
        self.__size = size
    def setmerk(self,merk):
        self.__merk = merk

    #getter
    def getsize(self):
        return self.__size
    def getmerk(self):
        return self.__merk